azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -580,6 +580,18 @@ func resourceKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"auto_upgrade_channel": {
```

Comment:
would this make more sense as
```suggestion
			"automatic_channel_upgrade": {
```

Reply:
yes agree - makes more sense.